support/module/packaged/modules.json

```diff
@@ -13,8 +13,8 @@
 			"integrity": "sha512-h1wM7mE/cgpk2n1YOYBNMNBQq5ViLxil6+GRnVqSaFB3LY3ubGSZVkZQ5rzC74anLlNXJVpsH/bP98AQ6zzXbA=="
 		},
 		"ti.map": {
-			"url": "https://github.com/appcelerator-modules/ti.map/releases/download/ios-3.0.1/ti.map-iphone-3.0.1.zip",
-			"integrity": "sha512-vD9DLBQgRpmkt8FBbEzRKWKFuJkUTwjwK2ijvRTR9lebGmsgRa9uA2uOLltR8w27b4mbtREZowAEYcVvkQ01Dw=="
+			"url": "https://github.com/appcelerator-modules/ti.map/releases/download/ios-3.0.2/ti.map-iphone-3.0.2.zip",
+			"integrity": "sha512-44NB1WsujEcW5OOsQPCUNY6PTJ31jmLoKFU7vA5V4Iu6vAgy4tEuO1n6Rio1LEa4Is23LMlajjeZW38J4XXn2w=="
```

Comment:
Hey there @sgtcoolguy! I ran `node scons.js modules-integrity` and manually copied the new SHA-hash from the JSON output of the console and pasted it in here. Is that the correct way to go? Looks a bit manual to me.